apidoc/Titanium/UI/ScrollView.yml

```diff
@@ -525,9 +525,11 @@ name: ScrollViewAnimationProperties
 summary: |
     A simple object for specifying the animation properties when scrolling the view.
 platforms: [iphone, ipad, android]
-since: '6.2.0'
+since: '6.1.0'
 properties:
   - name: animated
     summary: When set to `true` it will scroll smoothly to the destination.
+    description: When set to `true` it will scroll smoothly to the destination. 
+        On iOS the default value is `true`.
```

Comment:
Replace with:
```yml
description: |
    When set to `true` it will scroll smoothly to the destination. 
    On iOS the default value is `true`.
```